tap_snowflake/__init__.py

```diff
@@ -97,6 +97,9 @@ def schema_for_column(c):
         result.type = ['null', 'string']
         result.format = 'binary'
 
+    elif data_type == 'object':
```

Comment:
I'm wondering if this should use a mapping for `SEMI_STRUCTURED`. _Object_ is one of three types: https://docs.snowflake.com/en/sql-reference/data-types-semistructured.html.

I think they can all be handled very similarly here.

Reply:
I know that it works for object type but I dont think this workflow generally handles variant properly. Variants can hold parquet and XML too which would throw an exception when we try to json.loads(xml_data)